src/frontend/github-client-middleware.ts

```diff
@@ -16,7 +17,14 @@ export default (octokitApp: App): RequestHandler => (req: Request, res: Response
 	res.locals.client = GithubAPI({
 		auth: octokitApp.getSignedJsonWebToken()
 	});
-	res.locals.isAdmin = isAdmin(res.locals.github, req.log);
+
+	if (res.locals.jiraHost && await booleanFlag(BooleanFlags.CALL_IS_ADMIN_AS_APP, true, res.locals.jiraHost)){
+		req.log.info(`using app-authenticated github client for jira host ${res.locals.jiraHost}`);
+		res.locals.isAdmin = isAdmin(res.locals.client, req.log);
+	} else {
+		req.log.info(`using user-authenticated github client for jira host ${res.locals.jiraHost}`);
+		res.locals.isAdmin = isAdmin(res.locals.github, req.log);
+	}
```

Comment:
This changes the client used in the `isAdmin()` function from a user-authenticated client to the app-authenticated client. Hoping this might fix a customers issues with IP allowlist blocking, because they currently see some calls going through and some being blocked by the allowlist...